widget/windows/nsWindow.cpp

```diff
@@ -3670,6 +3671,11 @@ nsWindow::GetLayerManager(PLayerTransactionChild* aShadowManager,
   RECT windowRect;
   ::GetClientRect(mWnd, &windowRect);
 
+  if (CreateWebrenderLayerManager()) {
```

Comment:
So we are creating WRLM every time we call `nsWindow::GetLayerManager()`? Sounds fishy